pkgs/development/tools/gettext-tools/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv, lib, fetchurl, libiconv, xz, gettext-runtime, libtextstyle,
+  autoreconfHook }:
+
+
+let common = import ../../libraries/gettext/common.nix { inherit fetchurl; };
+in
+stdenv.mkDerivation rec {
+  pname = "gettext-tools";
+  inherit (common) version src;
+
+  sourceRoot = "${common.pname}-${common.version}/${pname}/";
+
+  patches = [
+    ./gettext.git-2336451ed68d91ff4b5ae1acbc1eca30e47a86a9.patch
+
+    # TODO when upgrading from 0.20.1
+    #
+    #   - remove the this patch (this is an updated backport from upstream)
+    #   - remove the autoreconfHook
+    #   - remove the 'cp ../libtextstyle/m4/libtextstyle.m4 gnulib-m4/' from postPach
+    ./restore_the_ability_to_build_gettext-tools_separately.patch
+  ];
+
+  outputs = [ "out" "man" "doc" "info" ];
+
+  configureFlags = [
+    "--disable-csharp"
+    "--with-xz"
+    "--with-installed-libtextstyle"
+  ];
+
+  postPatch = ''
+   cp ../libtextstyle/m4/libtextstyle.m4 gnulib-m4/
+   substituteInPlace projects/KDE/trigger --replace "/bin/pwd" pwd
+   substituteInPlace projects/GNOME/trigger --replace "/bin/pwd" pwd
+   substituteInPlace src/project-id --replace "/bin/pwd" pwd
+  '' + lib.optionalString stdenv.hostPlatform.isCygwin ''
+    sed -i -e "s/\(cldr_plurals_LDADD = \)/\\1..\/gnulib-lib\/libxml_rpl.la /" src/Makefile.in
+    sed -i -e "s/\(libgettextsrc_la_LDFLAGS = \)/\\1..\/gnulib-lib\/libxml_rpl.la /" src/Makefile.in
+  '';
+
+  nativeBuildInputs = [
+    xz
+    xz.bin
```

Comment:
Does not this already get picked up from xz?

Reply:
Yes it should. And `xz` is in stdenv so I should be able to drop this explicit dependency completely (both in gettext-runtime and gettext-tools). I am rebuilding in order to make sire nothing gets broken.